ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -1381,3 +1381,29 @@ def add_dns_zone(master, zone, skip_overlap_check=False,
                                     host.hostname + ".", '--a-rec', host.ip])
     else:
         logger.debug('Zone %s already added.', zone)
+
+
+def run_dnsserver_find(host, server_to_find, returncode=None):
```

Comment:
Same comment to function name and behavior as for `run_server_del`. Btw return code argument is not self-explanatory. It should e.g. `expected_returncode` or `dnsserver_present`.